tests/chart_tests/test_elasticsearch.py

```diff
@@ -304,11 +304,20 @@ def test_nginx_es_index_pattern_defaults(self, kube_version):
 
         assert len(docs) == 1
         doc = docs[0]
-        es_index = doc["data"]["nginx.conf"]
         assert doc["kind"] == "ConfigMap"
-        assert "fluentd.$remote_user.*/$1" in es_index
 
-    def test_nginx_es_index_pattern_with_sidecar_logging_enabled(self, kube_version):
+        nginx_config = " ".join(doc["data"]["nginx.conf"].split())
+        assert all(
+            x in nginx_config
+            for x in [
+                "location ~* /_search$ { rewrite /_search(.*) /fluentd.$remote_user.*/_search$1 break;",
```

Comment:
Should we have a check for bulk and count also?

Reply:
Yes! I just committed them.